pkgs/tools/networking/go-shadowsocks2/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "go-shadowsocks2-${version}";
```

Comment:
I believe `pname` should work with `buildGoPackage` from looking at its source

Reply:
Thanks